### PR TITLE
add OWNCLOUD_INTEGRITY_CHECK_DISABLED var for testing

### PR DESCRIPTION
**Pull request by micbar**
# Description

make the testing setup cleaner.

### CODE REVIEW

File: v20.04/overlay/etc/templates/config.php [hunk 1]
```diff
@@ -355,6 +355,10 @@ function getConfigFromEnv() {
     $config['excluded_directories'] = explode(',', getenv('OWNCLOUD_EXCLUDED_DIRECTORIES'));
   }
 
+  if (getenv('OWNCLOUD_INTEGRITY_CHECK_DISABLED') != '') {
+    $config['integrity.check.disabled'] = getenv('OWNCLOUD_INTEGRITY_CHECK_DISABLED') == 'false';
```
Comment (jnweiger) [inline v20.04/overlay/etc/templates/config.php:359]:
```suggestion
    $config['integrity.check.disabled'] = getenv('OWNCLOUD_INTEGRITY_CHECK_DISABLED') == 'true';
```

The negation of a disable flag ... is ... naaah....

**Review by jnweiger — CHANGES_REQUESTED** [review]
please double check the inverted logic...

(Okay to drop some 10.3.0 rebuilds en-passant!)

File: v20.04/overlay/etc/templates/config.php (hunk 1 above)
Comment (micbar) [reply v20.04/overlay/etc/templates/config.php:359]: no no, the default should be `false`

Comment (micbar) [reply v20.04/overlay/etc/templates/config.php:359]: In english: the default case should have the integrity check enabled

**Review by jnweiger — APPROVED** [review]

**Pull request merged**